doc/run_your_own.md

```diff
@@ -0,0 +1,53 @@
+# Run your own Fritz
+
+Clone the repo from `GitHub`:
+```bash
+git clone --recursive https://github.com/fritz-marshal/fritz.git && cd fritz
+```
+
+All operations are started via the `fritz` script.
+Make sure the requirements to run it are met, e.g.:
+
```

Comment:
Maybe ask the user to start their own conda env here so they don't cross-pollute the installed python modules

Reply:
Added some text on that.